src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -35,7 +35,7 @@
 
 image_by_tag_type = CLIArgumentType(
     options_list=['--image', '-t'],
-    help="The name of the image. May include a tag in the format 'name:tag'."
+    help="The name of the image. May include a tag in the format 'name:tag'. Multiple --image or -t can be fed."
```

Comment:
This is not always the case, e.g., in delete path, `image_by_tag_type` only allows one parameter.

If you need to mention the second part, I'd recommend override the help message where the parameter supports append.